tools/kafka_sasl_ssl/generate_ssl_CA/ssl_one_way.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+# The example: bash ssl_one_way.sh kafka-0.tigergraph.com ~/SSL_one_way tiger123
+# The example password: tiger123
+# The example server host name: kafka-0.tigergraph.com
+# The certificate generation path: ~/SSL_one_way
+
+if [ $# -eq 3 ]; then
+  server_hostname=$1
+  generate_root=$2
+  pass=$3
+else
+  echo "Error in parameter. Please check."
+  echo "e.g. bash ssl_one_way.sh server_hostname generate_root password"
+  exit 1
+fi
+
+env_prepare(){
+  # install java
+  if ! which java > /dev/null 2>&1; then
+    echo "start install openjdk."
+    sudo yum update -y > /dev/null 2>&1
+    sudo yum install -y java-1.8.0-openjdk > /dev/null 2>&1
+    echo "install openjdk-1.8.0 successfully."
+  else
+    java_version=$(java -version 2>&1 | sed '1!d' | sed -e 's/"//g' | awk '{print $3}'| cut -d_ -f1)
+    if [[ $java_version != "1.8.0" ]];then
+      echo "start upgrade java."
+      rpm -qa | grep java | sudo xargs rpm -e --nodeps
+      sudo yum update -y > /dev/null 2>&1
+      sudo yum install -y java-1.8.0-openjdk > /dev/null 2>&1
```

Comment:
1. don't do update
2. need to check OS: /Users/wonderdong/product/src/er/offline_install/installer/normal_install/utils/os_utils
3. need to check install tool failure